src/main/java/cat/udl/eps/entsoftarch/webingogeiapi/domain/UriEntity.java

```diff
@@ -46,4 +47,6 @@ public String getUri() {
     public boolean isNew() {
         return version == null;
     }
+
+    public abstract void setStartAt(ZonedDateTime now);
```

Comment:
Not needed, makes all derived entities have to implement this method, even if not required!